<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>551</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tên:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnName">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>100</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tính đêm bắt đầu từ:</string>
     </property>
    </widget>
    <widget class="QSlider" name="sld1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>170</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnSld1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Đến:</string>
     </property>
    </widget>
    <widget class="QSlider" name="sld2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnSld2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>220</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>290</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Giá ngày:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spb1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>290</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>290</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Giá đêm:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spb2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spb4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>360</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>360</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Giá phụ thu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>420</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Phụ thu từ:</string>
     </property>
    </widget>
    <widget class="QSlider" name="sld3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>430</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnSld3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>420</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>480</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Đến:</string>
     </property>
    </widget>
    <widget class="QSlider" name="sld4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>490</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnSld4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>480</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Giá vé tháng:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spb3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>360</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Loại phương tiện:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbbVehicle">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>281</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Xe máy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ô tô</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>551</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnOK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Đồng ý thêm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOKUpdate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Đồng ý sửa</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblMessage">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnUpdate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Sửa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDelete">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Xóa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Hủy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUse">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="text">
      <string>Sử dụng</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>331</width>
      <height>541</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Tên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Loại xe</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trạng thái</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>640</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Serif</family>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
